src/app/portlets/content-types/fields/content-type-fields-properties-form/content-type-fields-properties-form.component.ts

```diff
@@ -141,8 +166,10 @@ export class ContentTypeFieldsPropertiesFormComponent extends BaseComponent impl
             this.handleDisabledIndexed(true);
         }
 
-        checkbox.valueChanges.subscribe((res) => {
-            checkbox === this.form.get('unique') ? this.handleUniqueValuesChecked(res) : this.setIndexedValueChecked(res);
+        checkbox.valueChanges.subscribe(res => {
```

Comment:
Please check what's happening with with your prettier looks like is not taking the config.